pandas/tests/test_lib.py

```diff
@@ -39,6 +40,11 @@ def test_fast_unique_multiple_list_gen_sort(self):
         out = lib.fast_unique_multiple_list_gen(gen, sort=False)
         tm.assert_numpy_array_equal(np.array(out), expected)
 
+    def test_fast_unique_multiple_unsortable_runtimewarning(self):
+        idx = pd.MultiIndex.from_product([[1, pd.Timestamp("2000")], ["foo", "bar"]])
```

Comment:
can you target this test more specifically at the affected function?  maybe pass `np.array([Timestamp(...), "foo"])`?

It's not a rule/policy, but ideally the intra-pandas dependencies of test_lib would be limited to the dependencies of `lib`.